odd-platform-ui/src/components/DataQuality/DataQualityFilters/DataQualityFilters.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Grid, Typography } from '@mui/material';
+import { Button } from 'components/shared/elements';
+import * as S from './DataQualityFiltersStyles';
+import { NamespaceFilter } from './FilterItem/NamespaceFilter';
+import { DatasourceFilter } from './FilterItem/DatasourceFilter';
+import { OwnerFilter } from './FilterItem/OwnerFIlter';
+import { TitleFilter } from './FilterItem/TitleFilter';
+import { TagFilter } from './FilterItem/TagFilter';
+import { useDataQualityContext } from '../DataQualityContext/DataQualityContext';
+
+export const DataQualityFilters: FC = () => {
+  const { t } = useTranslation();
+  const { updateFilter } = useDataQualityContext();
+
+  function clearTablesFilters() {
+    updateFilter('deNamespaceIds', []);
+    updateFilter('deDatasourceIds', []);
+    updateFilter('deOwnerIds', []);
+    updateFilter('deTitleIds', []);
+    updateFilter('deTagIds', []);
+  }
+
+  function clearTestsFilters() {
+    updateFilter('namespaceIds', []);
+    updateFilter('datasourceIds', []);
+    updateFilter('ownerIds', []);
+    updateFilter('titleIds', []);
+    updateFilter('tagIds', []);
+  }
+
+  return (
+    <S.Container>
```

Comment:
You can wrap the filter inputs into react form and sync url state with forms' state but it's not necessary. Standalone filter inputs that each one sets their own part of url is fine too

Reply:
https://react-hook-form.com/docs/useform